diff --git a/node_modules/homeassistant/lib/HomeAssistant.js b/node_modules/homeassistant/lib/HomeAssistant.js
index 22c4068..d55f447 100644
--- a/node_modules/homeassistant/lib/HomeAssistant.js
+++ b/node_modules/homeassistant/lib/HomeAssistant.js
@@ -1,5 +1,13 @@
 'use strict';
 
+const camera = require('./Camera');
+const events = require('./Events')
+const history = require('./History')
+const logs = require('./Logs')
+const services = require('./Services')
+const states = require('./States')
+const templates = require('./Templates')
+
 class HomeAssistant {
   constructor(auth) {
     auth = auth || {};
@@ -24,10 +32,13 @@ class HomeAssistant {
       return obj;
     }, this);
 
-    ['Camera', 'Events', 'History', 'Logs', 'Services', 'States', 'Templates'].reduce((obj, id) => {
-      obj[id.toLowerCase()] = require(`./${id}`)(apiConfig);
-      return obj;
-    }, this);
+    this.camera = camera(apiConfig);
+    this.events = events(apiConfig);
+    this.history = history(apiConfig);
+    this.logs = logs(apiConfig);
+    this.services = services(apiConfig);
+    this.states = states(apiConfig);
+    this.templates = templates(apiConfig);
   }
 }
 
